src/modules/character.rs

```diff
@@ -69,6 +82,25 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
     Some(module)
 }
 
+fn is_root_euid(context: &Context) -> bool {
+    let root_uid: usize = 0;
+    let euid: Option<usize> = match context.exec_cmd("id", &["-u"]) {
+        Some(output) => match output.stdout.trim().parse() {
+            Ok(euid) => Some(euid),
+            Err(e) => {
+                log::warn!("Error parsing {} with error {}", output.stdout, e);
+                None
+            }
+        },
+        None => None,
+    };
+
+    match euid {
+        Some(euid) => euid == root_uid,
+        None => false,
+    }
+}
```

Comment:
You should reuse `is_root_user()` from the `username` module here (`src/modules/username.rs`).

Reply:
Thank you for your feedback! I've implemented your suggestion